tests/lib/rules/all.js

```diff
@@ -110,3 +110,34 @@ ruleTester.run('_', rules.concat, {
   ],
   invalid: []
 });
+
+ruleTester.run('_.isUndefined', rules['is-undefined'], {
+  valid: [
+    '2 === undefined'
+  ],
+  invalid: [{
+    code: '_.isUndefined(2)',
+    errors: ['Consider using the native value === undefined']
+  },{
+    code: '_(2).isUndefined()',
+    errors: ['Consider using the native value === undefined']
+
+  }]
+});
+
+/*This is to make sure that You-Dont-Need-Lodash can handle the 
+evaluation of nested functions that had caused an error noted in the comments of
+Pull Request #219*/
+for (let rule in rules) {
```

Comment:
Is testing one rule enough? If so, testing one is easier to read and maintain.

Reply:
@stevemao I wanted to be thorough, but since all of it passes through the same line, I suppose I could just do it for one rule.

Reply:
@stevemao Fixed it. I've also made the label much more obvious in the test output.